cvmfs/util/posix.cc

```diff
@@ -1918,30 +1919,30 @@ bool ManagedExec(const std::vector<std::string>  &command_line,
     // grand father
     pid_grand_child = getpid();
     failed = ForkFailures::kSendPid;
-    pipe_fork.Write(&failed, sizeof(failed));
-    pipe_fork.Write(pid_grand_child);
+    pipe_fork.Write<ForkFailures::Names>(failed);
+    pipe_fork.Write<pid_t>(pid_grand_child);
```

Comment:
Just a note: you can spell out the template parameter but you don't have to, templated functions and methods usually figure it out themselves.

Reply:
i know. i just thought it would be nicer for reading as a developer what parameters are actually send into the pipe